File: Application/Services/Operations/ProductServices/Dtos/EquipamentGridDto.cs
using System;
using System.Collections.Generic;
using Application.Services.Operations.Main.Companies.Dtos;

namespace Application.Services.Operations.ProductServices.Dtos
{
    public class EquipamentGridDto
    {
        public int ProductId { get; set; }
        public string Name { get; set; }
        public string Manufacturer { get; set; }
        public string Segment { get; set; }
        public string Model { get; set; }
        public string Description { get; set; }
        public int Quantities { get; set; }
    }
}

File: Views/src/components/product/dtos/equipament-grid-dto.ts
import { ProductDto } from "./product-dto";

export class EquipamentGridDto {
  productId:number;
  name: string;
  manufacturer: string;
  segment: string;
  model: string;
  description: string;
  btnDisabled: boolean;
  length: number;
  // entityComplete:ProductDto;
}
